specification/schemas/Organization.json

```diff
@@ -23,6 +23,37 @@
         "iban": {
           "$ref": "#/components/schemas/Iban"
         },
+        "language": {
+          "type": "enum",
+          "enum": [
+            "english"
+          ],
+          "example": "english"
+        },
+        "payment_method": {
+          "type": "enum",
+          "enum": [
+            "sepa_direct_debit",
+            "on_credit",
+            "prepayment"
+          ],
+          "example": "prepayment"
+        },
+        "vat_number": {
+          "type": "string",
+          "example": "123456789MVA"
+        },
+        "payment_term": {
+          "type": "integer",
```

Comment:
There is no integer (or at least not officially) in json schema and this should be `number`.